opal/templates/patient_lists/layouts/spreadsheet_list_base.html

```diff
@@ -3,27 +3,109 @@
 
 <div class="container-fluid content-offset">
 
+  {% block heading %}
```

Comment:
I see what you've done here - this is what is in the current list..

We might prefer to have naming convention consistency with the other two lists though, and use `  {% block list_actions %}`.

Reply:
(that means deleting the `spreadsheet_list_actions` block as well obv.)

Reply:
so is `heading` superfluous?
can it be removed,  and change `spreadsheet_list_actions` to `list_actions`?
I'm not 100% clear on the required end state